postgresql/spoon.me

```diff
@@ -0,0 +1,54 @@
+##
+# Licensed under the Apache License, Version 2.0
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Setup meta
+meta title="postgresql"
+meta namespace="postgresql" 
+meta name="postgresql"
+
+#Basic dependencies and tools
+from clean,microsoft/vcredist:2008
```

Comment:
the vcredist thing in clean is a hack right?

so ultimately it would be removed. in that case, shouldn’t scripts that actually need it keep it in? (not sure which is the case here)

On Jun 25, 2015, at 10:42 PM, pgalisz <notifications@github.com<mailto:notifications@github.com>> wrote:


In postgresql/spoon.me<https://github.com/spoonapps/spoonme/pull/47#discussion_r33329929>:

> @@ -0,0 +1,54 @@
> +##
> +# Licensed under the Apache License, Version 2.0
> +# http://www.apache.org/licenses/LICENSE-2.0
> +
> +# Setup meta
> +meta title="postgresql"
> +meta namespace="postgresql"
> +meta name="postgresql"
> +
> +#Basic dependencies and tools
> +from clean,microsoft/vcredist:2008


vcredist is implicit dependency, so it is not needed

—
Reply to this email directly or view it on GitHub<https://github.com/spoonapps/spoonme/pull/47/files#r33329929>.